content/en/docs/reference/command-line-tools-reference/feature-gates/ClusterTrustBundle.md

```diff
@@ -10,4 +10,13 @@ stages:
     defaultValue: false
     fromVersion: "1.27"
```

Comment:
Need to add beta details?

Reply:
Rebased to get this from the current main.